### DISCUSSION TITLE
Problem with Chia on Debian 10 PPC64

### DISCUSSION THREAD

**Discussion by WMark77** [Chia-Network/chia-blockchain#3995] [General]
Hello.

I'd like to run Chia on my IBM POWER system with Debian PowerpC (Big-Endian), but I can't install it.

Operating system details:
Linux 5.10.0-6-powerpc64 #1 SMP Debian 5.10.28-1 (2021-04-09) **ppc64** GNU/Linux

Installation (as root) steps:

1. mkdir chia  ; cd chia (OK)
2. python3.9 -m venv venv  (OK)
3. source venv/bin/activate  (OK)
4. (venv): pip install --upgrade pip  (OK)
5. pip install chia-blockchain==1.1.4 (ERROR):

... several packages are successfully installed/compilled ... 
... issue happens at **clvm_rs** step:
  
Downloading clvm_rs-0.1.7.tar.gz (470 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... error
    ERROR: Command errored out with exit status 1:

    ... **maturin failed**
      Caused by: **The architecture powerpc is not supported** ...
...
ERROR: No matching distribution found for clvm-rs==0.1.7

How can I solve that?

Thanks in advance.

**Comment by lvcivs** [comment]
Is ppc64 even a supported platform?

**Reply by WMark77** [reply]
I guess so because its installation (or part of it) can be compiled.

**Comment by lichaoyzf** [comment]
同样的问题
